app/templates/gentelella/users/events/index.html

```diff
@@ -91,7 +91,15 @@
     <script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/bootstrap-progressbar/bootstrap-progressbar.min.js') }}"></script>
     <script type="text/javascript">
-        $('.table').DataTable();
+    $('.table').DataTable( {"columnDefs": [
```

Comment:
@Himanshi-Khandelwal the alignment is still not proper here. You could have just copy pasted what i gave above ...